pkg/services/accesscontrol/models.go

```diff
@@ -102,7 +102,7 @@ const (
 	ScopeUsersAll = "users:*"
 
 	// Settings scope
-	ScopeSettingsAll = "settings:**"
```

Comment:
This scope is used for fixed roles and is required to have settings:**

Reply:
we're (possibly) moving to a simple prefix match so that we can translate it simple into SQL.  so everything is now **, effectively

Reply:
We should probably add a note about this in the docs + changelog in case someone have used `settings:**` in a custom role.

Reply:
I commented this before reading you design doc so this make sense